app/src/androidTest/assets/features/publicPrivateKeyGeneration.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Public and private key generation
+  As a user
+  I want to have the app generate a public/private key pair when I use it for the first time
+  So that I don't have to generate a pair myself
```

Comment:
I think our definitions don't comply indeed 😛 We don't use the key when it is exchanged, we could exchange whatever we want (for that matter, we will probably exchange the name of the user as well).

Think about it like this, we're trying to exchange random strings of text. These strings just happen to be public keys...